.github/workflows/backend.yml

```diff
@@ -104,7 +104,7 @@ jobs:
                 ports:
                     - 5432:5432
             clickhouse:
-                image: clickhouse/clickhouse-server:23.5.1.3174
+                image: clickhouse/clickhouse-server:23.11.2.11
```

Comment:
should we just use latest as we do with all other images? our clickhouse cloud is always upgraded to latest anyways

Reply:
sounds good, especially since this is just for testing